.github/actions/cilium-config/action.yml

```diff
@@ -63,7 +69,10 @@ runs:
             --helm-set=hubble.relay.image.useDigest=false \
             --helm-set=hubble.eventBufferCapacity=65535 \
             --helm-set=bpf.monitorAggregation=none \
-            --helm-set=authentication.mutual.spire.enabled=true \
+            --helm-set=cluster.name=default \
+            --helm-set=authentication.mutual.spire.enabled=${{ inputs.mutual-auth }} \
+            --helm-set=bpfClockProbe=${{ inputs.bpf-clock-probe }} \
+            --helm-set=cni.uninstall=false \
```

Comment:
Wait, why is this false? I read the PR description but if our intention is to replicate user-installations we should change defaults to accommodate our tests.

Reply:
My reading of the flag is that in most test cases it's a noop. But yeah, let's make it configurable.